[T13-4] CAPTracker

jeremiah-ang opened the pull request:
@alexkmj 
@josephambe
@jeremyyew 
@BugEyedBug

Pull request for [T13-4] CAPTracker

parvathy-sudhir-pillai commented on docs/AboutUs.adoc:
```diff
@@ -4,53 +4,52 @@
-image::damithc.jpg[width="150", align="left"]
-{empty}[http://www.comp.nus.edu.sg/~damithch[homepage]] [https://github.com/damithc[github]] [<<johndoe#, portfolio>>]
+=== Alex Koh
+image::alexkmj.jpg[width="150", align="left"]
```
Images except Jeremiah's not visible. 
The portfolio links of other developers give a 404 error.

parvathy-sudhir-pillai commented on docs/UserGuide.adoc:
```diff
@@ -12,19 +12,19 @@ ifdef::env-github[]
 == Introduction
 
-AddressBook Level 4 (AB4) is for those who *prefer to use a desktop app for managing contacts*. More importantly, AB4 is *optimized for those who prefer to work with a Command Line Interface* (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB4 can get your contact management tasks done faster than traditional GUI apps. Interested? Jump to the <<Qui ...
+CAPTracker is for those students who prefer to use a desktop app for calculating and managing their CAP. More importantly CAPTracker is *optimized for those who prefer to work with a Command Line Interface* (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, CAPTracker is the ideal application to calculate your current CAP, and predict what grades you  ...
```
Good job updating the user guide

parvathy-sudhir-pillai commented on docs/DeveloperGuide.adoc:
```diff
@@ -533,360 +533,97 @@ a. Include those libraries in the repo (this bloats the repo size) +
+
+=== Use case:  View grades needed to achieve CAP goal (based on previous grades)
+  . Use-Case-B.1.
+  . User enters CAP goal.
```
System does not respond after entering the CAP goal?

parvathy-sudhir-pillai commented on docs/DeveloperGuide.adoc:
```diff
@@ -533,360 +533,97 @@ a. Include those libraries in the repo (this bloats the repo size) +
+=== Use case:  Save entered module
+  . User enters some modules
+  . System saves the modules
+  . User restarts the application
```
The system can show the saved modules only after restart?

parvathy-sudhir-pillai commented on docs/DeveloperGuide.adoc:
```diff
@@ -533,360 +533,97 @@ a. Include those libraries in the repo (this bloats the repo size) +
-
-_{More to be added}_
+=== Use case: Calculate predicted CAP
+  . User enters all modules - modules with grades (previous or target), modules without grades (unknown).
```
Should be done in a loop. Refer to the notation for writing loops in use cases and update.

parvathy-sudhir-pillai commented on docs/DeveloperGuide.adoc:
```diff
@@ -533,360 +533,97 @@ a. Include those libraries in the repo (this bloats the repo size) +
+  . System saves the modules
+  . User restarts the application
+  . User list entered modules
+  . System displays saved modules
```
Try to follow the use-case notations as given in the example one provided by AddressBook with MSS, Extensions etc.

parvathy-sudhir-pillai reviewed (commented):
Overall a good effort.
 
Suggestions to improve further:
* All developers to update their respective portions of user and developer guides.
* Start to make changes in java files.

parvathy-sudhir-pillai commented on README.adoc:
```diff
@@ -15,19 +14,12 @@ ifndef::env-github[]
-** A more sophisticated GUI that includes a list  panel and an in-built Browser.
-** More test cases, including automated GUI testing.
-** Support for _Build Automation_ using Gradle and for _Continuous Integration_ using Travis CI.
+* CAPTracker is for those students who prefer to use a desktop app for calculating and managing their CAP. More importantly CAPTracker is optimized for those who prefer to work with a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, CAPTracker is the ideal application to calculate your current CAP, and predict what grades you  ...
```
"Ideal" CAP seems a bit unrealistic. May be "target" CAP is a better word.

parvathy-sudhir-pillai replied:
AB-4 should be part of acknowledgements

parvathy-sudhir-pillai commented on docs/UserGuide.adoc:
```diff
@@ -46,177 +46,81 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
-* `edit 2 n/Betsy Crower t/` +
-Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
+* `edit CS2103 CS1231 2 1` +
+Edits the module code and year/semester of `CS2103` module to be `CS1231` and `2 1` respectively.
```
"year/semester" or "year and (or) semester"?

parvathy-sudhir-pillai commented on docs/AboutUs.adoc:
```diff
@@ -4,53 +4,52 @@
-image::damithc.jpg[width="150", align="left"]
-{empty}[http://www.comp.nus.edu.sg/~damithch[homepage]] [https://github.com/damithc[github]] [<<johndoe#, portfolio>>]
+=== Alex Koh
+image::alexkmj.jpg[width="150", align="left"]
```
Alex's image is still not visible

parvathy-sudhir-pillai replied:
Also, use pictures which show your face clearly @jeremiah-ang

parvathy-sudhir-pillai commented on docs/AboutUs.adoc:
```diff
@@ -4,53 +4,52 @@
-image::m133225.jpg[width="150", align="left"]
-{empty}[http://github.com/m133225[github]] [<<johndoe#, portfolio>>]
+=== Jeremiah Ang
+image::jeremiah-ang.png[width="150", align="left"]
```
Suggest to add a photo where face is not too small to identify you

parvathy-sudhir-pillai commented on docs/DeveloperGuide.adoc:
```diff
@@ -533,360 +533,98 @@ a. Include those libraries in the repo (this bloats the repo size) +
-
-*Target user profile*:
+== Target User Profile
+- A NUS Student who has a need to keep track of current CAP, calculate expected CAP, and grades required to achieve desired CAP.
```
'An' NUS student

parvathy-sudhir-pillai commented on docs/DeveloperGuide.adoc:
```diff
@@ -533,360 +533,98 @@ a. Include those libraries in the repo (this bloats the repo size) +
-
-_{More to be added}_
+*Must-Have*
+
```
Suggestion to do a spellcheck and grammar check

parvathy-sudhir-pillai commented on docs/DeveloperGuide.adoc:
```diff
@@ -533,360 +533,98 @@ a. Include those libraries in the repo (this bloats the repo size) +
+4.  As a user I can edit the marks I’ve entered previously so that I can
+update my CAP.
+5.  As a user, I can enter in a CAP goal so I can track my progress in
+my modules.
```
Seems too vague. How do you track progress with only grades? What do you track progress of?

parvathy-sudhir-pillai commented on docs/DeveloperGuide.adoc:
```diff
@@ -533,360 +533,98 @@ a. Include those libraries in the repo (this bloats the repo size) +
+5.  As a user, I can enter in a CAP goal so I can track my progress in
+my modules.
+6.  As a user, I can enter a CAP goal so that I can keep track of the
+progress of my course
```
What's the difference between story 5 and 6?

parvathy-sudhir-pillai reviewed (commented):
Overall:
Good effort on meeting all requirements for the v1.1 team component. 
Your metlife preview is not loading. Check log and correct the issue.
Suggestions below for improvements in the upcoming versions:

* Developer Guide
<img width="1030" alt="screen shot 2018-10-07 at 12 24 53 am" src="https://user-images.githubusercontent.com/9066342/46573454-20754a80-c9c8-11e8-804a-2b0f8863bfb4.png">

Add more use cases

* Other comments inline

parvathy-sudhir-pillai commented on src/main/java/seedu/address/logic/commands/CapCommand.java:
```diff
@@ -0,0 +1,25 @@
+import seedu.address.model.Model;
+
+/**
+ * Show CAP based on existing modules.
```
"Shows" (descriptive style)

parvathy-sudhir-pillai commented on src/main/java/seedu/address/logic/parser/GoalCommandParser.java:
```diff
@@ -0,0 +1,31 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
```
Can avoid unnecessary line breaks between import statements

parvathy-sudhir-pillai commented on src/main/java/seedu/address/logic/parser/GoalCommandParser.java:
```diff
@@ -0,0 +1,31 @@
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parse User Input
```
"Parses"

jeremiah-ang committed:
Fix ordering

jeremiah-ang committed:
remove trailing whitespace

jeremiah-ang committed:
refractor reuse constructor

Commit:
Added test for AddModuleCommandParser and utils to testing. Corrected some logic for checking equals of Module and AddModuleCommand

Commit:
Fixed complaints by Codacy

Commit:
Added test for TranscriptParser

Commit:
Fixed codacy complaints

Commit:
Added javadocs for ParserUtil

Commit:
Fixed checkstyle

josephambe committed:
Merge pull request #6 from CS2103-AY1819S1-T13-4/master

Update with upstream

amberkj committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-T13-4/main

jeremyyew committed:
Merge pull request #53 from jeremiah-ang/updateDisplayPicture

Resolve issues pointed out by TA

Commit:
Merge pull request #52 from jeremyyew/gui-layout

Gui layout

Commit:
Merge pull request #55 from jeremiah-ang/extractCapGoal

Extract cap goal

Commit:
Change tokenize to tokenizes

Commit:
Merge pull request #54 from alexkmj/feature-add

Added Parser for Module and Transcript

jeremiah-ang committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-T13-4/main into calculateTargetCap

# Conflicts:
#	src/main/java/seedu/address/model/Transcript.java

jeremiah-ang committed:
Add state to Grade

jeremiah-ang committed:
fix checkstyle

jeremiah-ang committed:
fix styles

Commit:
Merge pull request #9 from CS2103-AY1819S1-T13-4/master

Merge upstream

jeremiah-ang committed:
Add javadoc

Commit:
add routing logic

jeremyyew committed:
Display mock text

jeremyyew committed:
Merge pull request #64 from alexkmj/logic-router

Logic router for TranscriptParser

jeremyyew committed:
Display cap and recalculate on transcriptChangedEvent

jeremyyew committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-T13-4/main into gui-layout

amberkj committed:
Sample module data displays in two columns

jeremiah-ang committed:
add tests

jeremiah-ang committed:
Add test cases

jeremiah-ang committed:
fix style

jeremiah-ang committed:
Merge branch 'gradeState' into calculateCurrentCapWGradeState

jeremiah-ang committed:
Adjust and Target grade by point

Commit:
Merge pull request #37 from jeremiah-ang/calculateTargetCap

Implement Calculate target module grade in Transcript Class

jeremiah-ang committed:
Fix suggestions

jeremiah-ang committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-T13-4/main into gradeState

jeremiah-ang committed:
Added EMPTY VALUE to regex

jeremiah-ang committed:
Add TODO to resolve several issue in the future

jeremiah-ang committed:
Add tests

jeremiah-ang committed:
adjust test cases

Konglomerator committed:
Add RedoModuleCommand
Add UndoModuleCommand
Update UserGuide

jeremiah-ang committed:
Merge branch 'gradeState' into calculateCurrentCapWGradeState

# Conflicts:
#	src/main/java/seedu/address/model/module/Grade.java

jeremiah-ang committed:
Make calculation of target grade fire on either 1) changes in the list or 2) changes in cap goal

jeremiah-ang committed:
add tests

Commit:
Merge pull request #10 from CS2103-AY1819S1-T13-4/master

Merge upstream

jeremiah-ang committed:
Add Comments

jeremiah-ang committed:
Add tests

jeremiah-ang committed:
Trigger by update

Commit:
Annotation for alexkmj

jeremiah-ang committed:
Fix unintended changes

jeremiah-ang committed:
Fix styles

Commit:
Added annotation for jeremy

Commit:
Added collate folder

jeremyyew committed:
Merge pull request #69 from alexkmj/annotate

Annotate codes

jeremiah-ang committed:
Add adjust command

jeremiah-ang committed:
Implement functions to support Adjust module command

jeremiah-ang committed:
Add tests

jeremiah-ang committed:
Merge branch 'gradeState' into feature/command/adjust/adjustCommand

jeremiah-ang committed:
Merge gradeState

Konglomerator committed:
Add RedoModuleCommand
Add UndoModuleCommand
Update UserGuide

Commit:
Merge branch 'master' into calculateCurrentCapWGradeState

Commit:
Merge branch 'master' into feature/command/adjust/adjustCommand

Commit:
Merge branch 'master' into gradeState

jeremiah-ang committed:
Remove whitespace

jeremiah-ang committed:
Update docs

jeremyyew committed:
Merge pull request #62 from jeremiah-ang/gradeState

Add state to Grade

amberkj committed:
Developer Guide updated and Titles for UI columns done

amberkj committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-T13-4/main

amberkj committed:
Merge branch 'master' of https://github.com/josephambe/main

Commit:
Merge branch 'master' into calculateCurrentCapWGradeState

Commit:
Merge pull request #66 from jeremiah-ang/calculateCurrentCapWGradeState

Calculate Target Grade triggers

amberkj committed:
UI Titles for Module Pannels are in line

jeremyyew committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-T13-4/main into gui-layout

jeremyyew committed:
Add getTranscript, CurrentCap displays

jeremyyew committed:
Display both currentCap and capGoal

jeremyyew committed:
Fix checkstlye

parvathy-sudhir-pillai reviewed (commented):
### Feedback for v1.2

**User Guide**
* This command seems to be doing something different from the description. The command heading is compute CAP, whereas what it does is to set target CAP
![t13-1](https://user-images.githubusercontent.com/9066342/47263690-43ac0800-d539-11e8-8856-018aba7e93e3.png)

**Developer Guide**
* No significant updates since v1.2
* Not enough new/updated design descriptions and UML diagrams to give feedback on. Please try to add more soon.
* While drawing SDs, keep in mind that we discourage low level diagrams that span multiple components. Try to draw diagrams at multiple levels: one that shows component level interactions and others shown low level interactions within one components.

### Overall
Work on making the product user-testable soon, and catch up on documentation work. The progress at the moment seems a bit behind, at least w.r.t Documentation, than what is required at this stage 😟

Commit:
Merge pull request #11 from CS2103-AY1819S1-T13-4/master

Merge upstream

Konglomerator committed:
Add RedoModuleCommandTest
Add UndoModuleCommandTest

Commit:
Added delete module feature

jeremiah-ang committed:
refractor change names and signatures

Konglomerator committed:
Checkstyle fix

Commit:
Merge pull request #72 from BugEyedBug/ReplaceLegacyUndoRedo

Undo/Redo Command

jeremiah-ang committed:
Update comparison logic

jeremiah-ang committed:
Add tests

jeremiah-ang committed:
Merge branch 'feature/model/transcript/module/comparingModules' into feature/command/adjust/adjustCommand

# Conflicts:
#	src/main/java/seedu/address/model/Transcript.java

jeremiah-ang committed:
fix tests

Commit:
Add delete command

Commit:
Changed isSameModule logic and add methods to remove module

Commit:
Add tests

Commit:
changed javadocs in AddModuleCommandParser

Commit:
Changed documentation on add, delete, and list

jeremiah-ang committed:
Fix cap goal command doc

jeremiah-ang committed:
Add sequence diagrams

Commit:
Fixed codacy issues and added DeleteModuleCommandParserTest

Commit:
Fix checkstyle issues

Commit:
Merge branch 'master' into feature-delete

Commit:
added author annotation and modified ModuleTest

Commit:
modified test for UniqueModuleList and changed logic on setModule

Commit:
modified test for UniqueModuleList and changed logic on setModule

Commit:
Merge pull request #12 from CS2103-AY1819S1-T13-4/master

Merge upstream

Commit:
Code and Grade text automatically changes to uppercase

Commit:
Acknowledge SE-EDU

Commit:
Created EditModuleComand and EditModuleCommandParser

Commit:
Remove unused imports

Commit:
Fix eol bug

Konglomerator committed:
Rename getStorageSaveLocation() to getAddressStorageSaveLocation()
Rename getModel() to getAddressBookModel()
Add methods in TestApp to produce defensive copies of Transcript
Add helper methods in ModelHelper

Konglomerator committed:
Checkstyle fix

Konglomerator committed:
Checkstyle fix

jeremyyew committed:
Merge pull request #97 from BugEyedBug/SystemTests

System test setup

jeremyyew committed:
Merge pull request #95 from alexkmj/fix-acknowledgements

Acknowledge SE-EDU3/4 in README.adoc

jeremyyew committed:
Merge pull request #93 from alexkmj/improve-code

Module code and grade value set to uppercase

jeremiah-ang committed:
fire calculate CAP only on module change

Commit:
added hasMultipleInstance

jeremiah-ang committed:
Update Use case to follow MSS and Extensions

Commit:
Added hasMultipleInstances to Transcript, Model, ModelManager

Commit:
Added test

jeremyyew committed:
Merge pull request #98 from alexkmj/feature-hasmultipleinstance

Added hasMultipleInstance

Commit:
Merge pull request #79 from jeremiah-ang/feature/model/transcript/capGoalInterface

refractor CapGoal change names and signatures

jeremiah-ang committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-T13-4/main into feature/command/adjust/adjustCommand

Commit:
Merge pull request #13 from CS2103-AY1819S1-T13-4/master

Merge upstream

Commit:
Merge pull request #14 from alexkmj/master

Merge upstream

Commit:
Fix bugs

jeremiah-ang committed:
Allow user to enter code only or code+year+sem

jeremiah-ang committed:
add tests

Commit:
Merge pull request #99 from jeremiah-ang/documentation/usecase/improvewriteup

Update Use case to follow MSS and Extensions

jeremiah-ang committed:
Extract into method in Transcript to capture modules updated

Commit:
Merge pull request #78 from alexkmj/feature-delete

Added delete module command, Change logic for isSameModule

jeremiah-ang committed:
add Tests

jeremiah-ang committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-T13-4/main into feature/command/adjust/adjustCommand

# Conflicts:
#	src/main/java/seedu/address/logic/parser/TranscriptParser.java
#	src/main/java/seedu/address/model/module/Module.java

jeremiah-ang committed:
remove redundant code

jeremiah-ang committed:
add more tests

josephambe committed:
Returned to priorCircleBarState

josephambe committed:
Basic design for uncompleted modules list panel

josephambe committed:
Restructured capSummary Panel

josephambe committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-T13-4/main into priorCircleBarState

Commit:
Merge pull request #16 from alexkmj/master

Merge upstream

josephambe committed:
Updated style to pass tests

josephambe committed:
Further style updates

josephambe committed:
Updated style again

josephambe committed:
Removing unused methods and imports from ModuleListPanel

josephambe committed:
Remove unncessary Person methods

josephambe committed:
New ModuleCard tests

jeremiah-ang committed:
Improve tests

jeremiah-ang committed:
fix styles

jeremiah-ang committed:
update test

jeremiah-ang committed:
Fix tests

jeremiah-ang committed:
remove unused import

jeremiah-ang committed:
Update user guide

jeremiah-ang committed:
Implement interface for ReadOnlyTranscript

jeremiah-ang committed:
write test

jeremiah-ang committed:
fix styles

jeremiah-ang committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-T13-4/main into feature/model/modelManagerAPI

# Conflicts:
#	src/main/java/seedu/address/model/Transcript.java

jeremiah-ang committed:
refractor method name

josephambe committed:
refactorted GuiTestAssert tests

josephambe committed:
Xml AdaptedModule class created

Commit:
update command message

Co-Authored-By: jeremiah-ang <e0032087@u.nus.edu>

Commit:
reuse module

Co-Authored-By: jeremiah-ang <e0032087@u.nus.edu>

josephambe committed:
All tests pass, just style errors now

jeremiah-ang committed:
Update Model component of Developer Guide

jeremiah-ang committed:
Move filter functions from ReadOnlyTranscript to Model

josephambe committed:
BUILD SUCCESSFUL

josephambe committed:
Small changes from PR Quality Review

josephambe committed:
Commented out unused import statements PersonListPanelTest

josephambe committed:
Removed unused XML File tests in ModuleListPanelTest

josephambe committed:
Commented out unused field in ModuleListPanelTest

josephambe committed:
Fixing unused import statements. Again.

jeremyyew committed:
Merge pull request #88 from jeremiah-ang/documentation/userguide/fixGoalCommand

Fix cap goal command doc

jeremyyew committed:
Merge pull request #89 from jeremiah-ang/documentation/diagram/sequence/targetandcapcalculation

Add sequence diagrams for Target Grade and CAP calculation

Commit:
Merge pull request #112 from jeremiah-ang/documentation/devguide/model

Update Model component of Developer Guide

jeremiah-ang committed:
fix bug

Commit:
Modified tokenizer in ParserUtil. Deleted unrelated tests in ParserUtilTest. Enforce use of constants in typical module for standardisation

Commit:
Fix import issues

jeremyyew committed:
Separate cap values string and double

jeremyyew committed:
Annotate author

jeremyyew committed:
Update Storage section of Dev guide

jeremyyew committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-T13-4/main into jeremy-docs

jeremiah-ang committed:
update test with modules from TypicalModules

jeremiah-ang committed:
Remove unused codes

jeremiah-ang committed:
Fix checkstyle

jeremiah-ang committed:
refractor remove methods

Commit:
Merge pull request #113 from alexkmj/fix-tokenizer

Fix bug for ParserUtil.tokenizer and enforce use of constants in TestUtil.TypicalModules

jeremiah-ang committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-T13-4/main into feature/command/adjust/adjustCommand

jeremiah-ang committed:
Use static value

josephambe committed:
created a new test insetad of overridding

josephambe committed:
New getFilteredModuleList  test instead of replacing values

josephambe committed:
Justified removal of Index Test

josephambe committed:
Added missing JavaDoc comments to tests

jeremyyew committed:
Upate Storage diagram

josephambe committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-T13-4/main into priorCircleBarState

* 'master' of https://github.com/CS2103-AY1819S1-T13-4/main:
  Fix import issues
  Modified tokenizer in ParserUtil. Deleted unrelated tests in ParserUtilTest. Enforce use of constants in typical module for standardisation
  Update Model component of Developer Guide
  Add sequence diagrams
  Fix cap goal command doc

jeremyyew committed:
Update Storage docs and png

josephambe committed:
Updated UiClassDiagram.png and added a few lines on the UI component structure

jeremyyew committed:
Merge pull request #102 from josephambe/priorCircleBarState

Updated UI Features

jeremyyew committed:
Fix trailing whitespace

jeremyyew committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-T13-4/main into gui-layout

jeremyyew committed:
Obtain ReadOnlyTranscript from Logic rather than Transcript

jeremiah-ang committed:
Update API on model

jeremiah-ang committed:
fixed styles

jeremiah-ang committed:
Merge branch 'feature/model/modelManagerAPI' into feature/ui/panels/displayCorrectModules

jeremiah-ang committed:
Merge branch 'feature/command/adjust/adjustCommand' into feature/ui/panels/displayCorrectModules

# Conflicts:
#	src/main/java/seedu/address/model/Transcript.java
#	src/test/java/seedu/address/logic/commands/AddModuleCommandTest.java

jeremiah-ang committed:
Implement interface on Logic

jeremiah-ang committed:
Merge branch 'feature/model/modelManagerAPI' into feature/ui/panels/displayCorrectModules

jeremiah-ang committed:
link correct lists

Commit:
Merge pull request #17 from CS2103-AY1819S1-T13-4/master

Merge upstream

Commit:
Merge pull request #18 from alexkmj/master

Merge upstream

Commit:
Merge pull request #106 from jeremiah-ang/feature/model/modelManagerAPI

Implement ReadOnlyTranscript API

jeremyyew committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-T13-4/main into gui-layout

jeremiah-ang committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-T13-4/main into feature/command/adjust/adjustCommand

# Conflicts:
#	src/main/java/seedu/address/model/Transcript.java
#	src/test/java/seedu/address/logic/commands/AddModuleCommandTest.java

jeremiah-ang committed:
fix style

jeremyyew committed:
Working UI'

Commit:
Merge pull request #71 from jeremiah-ang/feature/command/adjust/adjustCommand

Adjust command

jeremiah-ang committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-T13-4/main into feature/ui/panels/displayCorrectModules

# Conflicts:
#	src/test/java/seedu/address/logic/commands/AddModuleCommandTest.java

jeremyyew committed:
Fix checkstyle

jeremyyew committed:
Merge pull request #126 from jeremiah-ang/feature/ui/panels/displayCorrectModules

Display correct modules for the two panels

Commit:
Merge pull request #76 from jeremyyew/gui-layout

Display Current CAP and CAP Goal

Commit:
Merge pull request #119 from jeremyyew/jeremy-docs

Update Developer Guide - Design - Storage

josephambe committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-T13-4/main into EditingUIDeveloperGuide

* 'master' of https://github.com/CS2103-AY1819S1-T13-4/main: (52 commits)
  Fix checkstyle
  Working UI'
  fix style
  link correct lists
  Implement interface on Logic
  fixed styles
  Update API on model
  Obtain ReadOnlyTranscript from Logic rather than Transcript
  Fix trailing whitespace
  Update Storage docs and png
  Upate Storage diagram
  Use static value
  refractor remove methods
  Fix checkstyle
  Remove unused codes
  update test with modules from TypicalModules
  Update Storage section of Dev guide
  Annotate author
  Separate cap values string and double
  fix bug
  ...

jeremiah-ang committed:
Update deserialiser

jeremiah-ang committed:
update tests

jeremiah-ang committed:
update capGoal

Commit:
Merge pull request #122 from josephambe/EditingUIDeveloperGuide

Editing UI developer guide

jeremiah-ang committed:
Update tests

Commit:
Edited UserGuide

Commit:
Merge pull request #19 from CS2103-AY1819S1-T13-4/master

Merge upstream

Commit:
Merge pull request #20 from alexkmj/master

Merge upstream

josephambe committed:
Updated size of cap goal text and rounded current cap to one decimal place

josephambe committed:
fixed formatting issues

Commit:
Fix delete module bug

Commit:
Merge branch 'feature-delete' of https://github.com/alexkmj/main into feature-delete

Commit:
Merge pull request #130 from alexkmj/fix-userguide

Update Userguide

Commit:
Made changes to edit command

Commit:
fix bug

Commit:
Addressed checkstyle issue

Commit:
Merge remote-tracking branch 'origin/master' into add-edit

Commit:
Added edit command

Commit:
Fix comparison bug in EditModuleCommand

Commit:
Fix routing for undo and redo

Commit:
Fix parsing bug

jeremyyew committed:
Merge pull request #136 from alexkmj/fix-commandrouting

Fix command routing for Undo and Redo

Commit:
Merge branch 'master' into feature-edit

Commit:
Remove build.gradle edit

Commit:
Added some tests for ParserUtil

Commit:
Added unit test

Commit:
add EOL for build.gradle

jeremiah-ang committed:
refractor code more readable

josephambe committed:
Removed dupliate round method and moved it to CapPanel class which deals more directly with the UI

josephambe committed:
Removed some sneaky white spaces which were failing styling tests

Commit:
Merge pull request #133 from alexkmj/feature-delete

Fix bugs in delete command

Commit:
Merge pull request #129 from jeremiah-ang/bug/target/notUpdating

target grades not updating

Commit:
Merge pull request #132 from josephambe/capGoalUiDisplay

Updated CAP Goal UI design

Commit:
Merge pull request #135 from alexkmj/feature-edit

Add Edit Command Feature

alexkmj commented:
Close this PR and create a new PR to resolve the build fail issue. [Refer here.](https://github.com/nus-cs2103-AY1819S1/forum/issues/118)

damithc commented:
Closed because Travis seems to be misbehaving
